docs/index.md

```diff
@@ -1,19 +1,19 @@
-
+NOMAD [[nomad-lab.eu](nomad-lab.eu){:target="\_blank"}] [1] is an open-source, community-driven data infrastructure that supports automated (meta)data extraction from a wide range of simulations, including ab initio and advanced many-body calculations, as well as molecular dynamics simulations. NOMAD also provides extensive customization capabilities to support experimental data. NOMAD allows users to store both standardized and custom complex workflows, which streamline data provenance storage and analysis, facilitating efficient curation of AI-ready datasets. This tutorial series will focus on recently developed workflow functionalities and utilities within the NOMAD infrastructure, with a step-by-step guide for storing a custom project workflow that contains tasks involving a variety of distinct data sources. Attendees can then use this knowledge to transform their day-to-day project data management, or even to interface with the NOMAD repository in a high-throughput fashion, opening improved discovery pipelines by leveraging the benefits of NOMAD’s comprehensive and FAIR-compliant data management system [2]. Attendees are welcome to simply watch the demonstration or to follow along on their laptops. For the latter, preparation instructions will be provided in advance below.
 
 [1] Scheidgen, M. et al., JOSS 8, 5388 (2023).
 
 [2] Scheffler, M. et al., Nature 604, 635-642 (2022).
 
+FAIRmat is present with several contributions at the DPG Spring Meeting of the Condensed Matter Section (SKM) 2025, [find locations and times here.](https://www.fairmat-nfdi.eu/events/fairmat-contribution-at-the-dpg-spring-meeting-of-the-condensed-matter-section-skm-2025/fairmat-contribution){:target="\_blank"}
```

Comment:
Suggested re-wording:

**Curious about more NOMAD features?**
You can find several other presentations, as well as a booth, here at the DPG Spring Meeting of the Condensed Matter Section (SKM) 2025.
For times and locations, [click here](https://www.fairmat-nfdi.eu/events/fairmat-contribution-at-the-dpg-spring-meeting-of-the-condensed-matter-section-skm-2025/fairmat-contribution){:target="\_blank"}.